eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -167,6 +167,34 @@
         }
       }
     },
+    {
+      "StaticConfigs": {
+        "Ubuntu22": {
```

Comment:
What's the name of the pipeline run corresponding to this?

Reply:
platform-matrix.json is used for the "cpp - <service> - ci" pipelines.